druntime/src/importc.h

```diff
@@ -109,6 +109,7 @@
 #if _MSC_VER
 //#undef _Post_writable_size
 //#define _Post_writable_size(x) // consider #include <no_sal2.h>
+#define _CRT_INSECURE_DEPRECATE(x)
```

Comment:
Why is this necessary? Shouldn't it be up to the user to decide whether to disable this warning?

Reply:
Not doing this turns on a lot of inline deprecated functions, and a blizzard of deprecation messages come out.